test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -2344,6 +2349,259 @@ public async Task ActorOrchestration_StringStoreWithCreateDelete(bool extendedSe
+                    {
+                        break;
+                    }
+                }
+
+                Assert.Equal(OrchestrationRuntimeStatus.Completed, status?.RuntimeStatus);
+                Assert.Equal("ok", status?.Output);
+
+                await host.StopAsync();
+            }
+        }
+
+        /// <summary>
+        /// End-to-end test which validates simple actor scenario.
+        /// </summary>
+        [Theory]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory + "_UnpublishedDependencies")]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task DurableActor_LockedIncrements(bool extendedSessions)
+        {
+            string[] orchestratorFunctionNames =
+            {
+                nameof(TestOrchestrations.LockedBlobIncrement),
+            };
+            using (var host = TestHelpers.GetJobHost(
+                this.loggerProvider,
+                nameof(this.DurableActor_LockedIncrements),
+                extendedSessions))
+            {
+                await host.StartAsync();
+
+                await EnsureBlobContainerExists("test");
+
+                var actorPlayingALock = new ActorId("Counter", Guid.NewGuid().ToString()); // does not matter what actor we use
+
+                // start two concurrent increment operations
+                // the lock should prevent incorrect interleavings
+
+                var client1 = await host.StartOrchestratorAsync(orchestratorFunctionNames[0], actorPlayingALock, this.output);
+                var client2 = await host.StartOrchestratorAsync(orchestratorFunctionNames[0], actorPlayingALock, this.output);
+                var client3 = await host.StartOrchestratorAsync(orchestratorFunctionNames[0], actorPlayingALock, this.output);
+
+                var status1 = await client1.WaitForCompletionAsync(TimeSpan.FromSeconds(Debugger.IsAttached ? 3000 : 15), this.output);
+                var status2 = await client2.WaitForCompletionAsync(TimeSpan.FromSeconds(Debugger.IsAttached ? 3000 : 15), this.output);
+                var status3 = await client3.WaitForCompletionAsync(TimeSpan.FromSeconds(Debugger.IsAttached ? 3000 : 15), this.output);
+
+                Assert.Equal(OrchestrationRuntimeStatus.Completed, status1?.RuntimeStatus);
+                Assert.Equal(OrchestrationRuntimeStatus.Completed, status2?.RuntimeStatus);
+                Assert.Equal(OrchestrationRuntimeStatus.Completed, status3?.RuntimeStatus);
+
+                var result = new int[] { (int)status1?.Output, (int)status2?.Output, (int)status3?.Output };
+                Array.Sort(result);
+
+                for (int i = 0; i < result.Length; i++)
+                {
+                    Assert.True(result[i] == i + 1);
+                }
+
+                await host.StopAsync();
+            }
+        }
+
+        /// <summary>
+        /// End-to-end test which validates simple actor scenario.

```

Comment:
It looks like the same summary is being used for multiple tests. It would be helpful if the summary could briefly explain what this test is specifically validating (e.g. which locking functionality, etc.)

Reply:
Definitely. I am adding descriptions.